.circleci/real_config.yml

```diff
@@ -1530,7 +1530,12 @@ jobs:
       - skip-if-github-only
       - run: apt-get update && apt-get -y install make
       - react-get-deps
-      - run: npx playwright install --with-deps && npx playwright install chrome
+      - run:
+          name: Install Playwright
+          command: |
+            cd ./webui/react
+            npx playwright install --with-deps && npx playwright install chrome
+            cd -
```

Comment:
**suggestion:** wdy think about making this a target in `webui/react/Makefile` such that you can just call `make -C web/react install-playwright` here?

Reply:
i was thinking about doing it, but the command is only used in ci (for local install, just `npx playwright install`), but i'll change it with make command